controllers/form.js

```diff
@@ -44,12 +44,20 @@ module.exports = function () {
   constructor.prototype = {
     events: {
       submit: 'closeForm',
-      close: 'closeForm'
+      close: 'closeForm',
+      'input focusin': 'stopFocus',
+      'input focus': 'stopFocus',
+      '[contenteditable]': 'stopFocus',
+      'select': 'stopFocus'
     },
 
     closeForm: function (e) {
       e.preventDefault();
       focus.unfocus().catch(_.noop);
+    },
+
+    stopFocus: function (e) {
+      e.preventDefault();
```

Comment:
this prevents mobile browsers from doing weird resizing things (separate from the zooming issue). Focusing actually works fine, and you can scroll through inputs as you'd expect (with native browser arrows, or by tapping other inputs).